apps/frontend/src/components/TimeCardPage/utils.tsx

```diff
@@ -27,9 +27,11 @@ export const createToast = (props: UseToastOptions) => {
 export const createNewComment = (
     user: UserSchema,
     type: CommentType,
-    content: string
+    content: string,
+    UUID: string = crypto.randomUUID()
```

Comment:
why are we doing this?

Reply:
The UUID is part of the comment schema and it doesn't autogenerate like I thought it did so nothing would actually save on the backend until I added that in.